### DISCUSSION TITLE
Mortal Kombat X

### DISCUSSION THREAD

**Discussion by skiffd** [elrumo/macOS_Big_Sur_icons_replacements#1208] [Icon request] [answered]
Can some one do MKX icon for big sure plz?

**Comment by cake03** [comment, marked as answer]
<img height="512" src="https://user-images.githubusercontent.com/80169337/122158137-bd7f2580-ce89-11eb-9972-599d7d4b64fe.png" alt="MK X" />
<h1> <a href="https://user-images.githubusercontent.com/80169337/122158137-bd7f2580-ce89-11eb-9972-599d7d4b64fe.png"> Download </a> </h1>

**Reply by skiffd** [reply]
Thx you :)
